app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,69 @@
+require 'pry'
+class TweetController < ApplicationController
+
+  get "/tweets" do
+    if logged_in?
+      @user = current_user
+      erb :"/posts/index"
+    else
+      redirect to "/login"
+    end
+  end
+
+  get "/tweets/new" do
+    if logged_in?
+      erb :"/posts/new"
+    else
+      redirect to "/login"
+    end
+  end
+
+  post "/tweet" do
+    if params[:content] != ""
+      tweet = Tweet.new(content: params[:content])
+      tweet.user_id = current_user.id
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.